x-pack/heartbeat/monitors/browser/synthexec/execmultiplexer.go

```diff
@@ -14,11 +14,11 @@ type ExecMultiplexer struct {
 	done         chan struct{}
 }
 
-func (e ExecMultiplexer) Close() {
+func (e *ExecMultiplexer) Close() {
```

Comment:
Sort of an unrelated change, but since we only ever use `*ExecMultiPlexer` that should really be the receiver. I hit a weird bug in some tests that this fixes.